pingen/controllers/main.py

```diff
@@ -50,14 +50,14 @@ def _get_letter_infos(self, json_content, document_uuid):
                 return node.get("attributes", {})
         return {}
 
-    def _get_emmited_date(self, json_content):
-        emmited_date = ""
+    def _get_emitted_date(self, json_content):
+        emitted_at = ""
         for node in json_content.get("included",{}):
             if node.get("type") == "letters_events":
                 attributes = node.get("attributes",{})
                 if attributes.get("code") == "transferred_to_distributor":
-                    emmited_date = attributes.get("emmited_at",{})
-        return emmited_date
+                    emitted_at = attributes.get("emitted_at", "")
```

Comment:
should we break here, as another node might set this to an empty string?

Reply:
Good catch. However, there shouldn't be another node with the same code, but it will at least avoid executing code once we found the date.